views/single.twig

```diff
@@ -26,7 +26,7 @@
 
 					<div class="feature-image">
 						{% set image = get_image(post.thumbnail()) %}
```

Comment:
get_image is done in the image.twig partial it shouldn't be needed here.

Reply:
updated

Reply:
updated

Reply:
updated